Makefile

```diff
@@ -13,9 +13,9 @@ init:
 	@pip install -r requirements-dev.txt
 
 lint:
-	flake8 moto
+	black --check moto/ tests/
 
-test: lint
```

Comment:
because we run it via travis and only in python 3.6 and up (black doesn't support older python versions.)